### PR TITLE
fix title draw on popup dialog

### PR DESCRIPTION
**Pull request by guruathwal**
- Fix: title display on the popup dialog after #2447.
~- Fix: progress bar not updating if progress is 100% after #2404.~
- Simplify the progress bar display code and remove redundant variables.
- cleanup and remove duplicate lines.

Fixes: #2476

### CODE REVIEW

File: TFT/src/User/Menu/PrintingMenu.c [hunk 1]
```diff
@@ -397,7 +397,7 @@ static inline void reDrawProgress(uint8_t prevProgress)
 {
   uint8_t nextProgress = getPrintProgress();
 
-  if (nextProgress > prevProgress)
+  if (nextProgress != prevProgress)
```
Comment (kisslorand) [inline TFT/src/User/Menu/PrintingMenu.c:400]: It surely won't fix it as progress must go upwards and as you seen in issue #2479 it stopped at 11%.

**Comment by Guilouz** [comment]
@bigtreetech Confirmed working, you can merge it.

**Comment by Guilouz** [comment]
@guruathwal  I spoke too quickly, issue for progress bar is still present.

IMG_3175

Printing is complete but the progress bar shows 39%.

**Comment by kisslorand** [comment]
@Guilouz 
You might want to check the fix for it: #2491.

**Pull request merged**